### PR TITLE
[Instrument Manager] updated module help text

### PR DESCRIPTION
**Pull request by knoll-alex**
new copy for the instrument manager module help text

#### Testing instructions

1. review copy
2. see screenshot for rendering

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576

Screenshot 2019-11-14 18 27 57

### CODE REVIEW

**Comment by driusan** [comment]
What does the "Needs Work" on this signify?

**Comment by christinerogers** [comment]
This PR has lost a level of useful detail about what the table columns actually mean. 
This is key for the admins who will be the only users of this module. 
the needs work label has been applied.

File: modules/instrument_manager/help/instrument_manager.md [hunk 1]
```diff
@@ -1,54 +1,15 @@
...
-## Viewing Installed Instruments
+**Note**: the *Table Valid* and *Pages Valid* columns can only be determined for LINST instruments (those created in the instrument builder), as there is no way to extract the “proper” values from instruments coded in PHP. The *Instrument Type* column displays whether the instrument was created in the instrument builder or coded in PHP. 
 
-Below the "Upload Instrument" section is a table displaying all
-instruments which are currently installed in LORIS. For each
-instrument it displays the columns:
+## Upload New Instrument
 
-*Instrument*: This provides the name of the instrument (as given
-in the `test_names` table).
```
Comment (christinerogers) [inline modules/instrument_manager/help/instrument_manager.md]: pls add back : ... as in the test names table

File: modules/instrument_manager/help/instrument_manager.md [hunk 2]
```diff
@@ -1,54 +1,15 @@
...
+If configured, the **Upload** tab will display an *Upload Instrument* section. Click **Browse** to select a LINST file from your computer. Click **Install Instrument** to install the instrument into LORIS. 
 
-*Instrument Type*: This contains either "PHP" or "Instrument Builder",
```
Comment (christinerogers) [inline modules/instrument_manager/help/instrument_manager.md]: please add back all this detail below.  pare down / re-style by all means, thanks!

File: modules/instrument_manager/help/instrument_manager.md [hunk 3]
```diff
@@ -1,54 +1,15 @@
 # Instrument manager
 
-## Uploading Instruments
+This module allows you to view the status of instruments that are installed in LORIS. If configured, the module will contain an **Upload** tab, where you can install new instruments created with the Instrument Builder. 
 
-The instrument manager provides the ability to view the status of
-instruments that are installed in LORIS and, if configured, the
-ability to install new instruments created with the instrument
-builder.
+## View Installed Instruments
```
Comment (christinerogers) [inline modules/instrument_manager/help/instrument_manager.md]: pls change to: `View Status of installed instruments`

Comment (knoll-alex) [reply modules/instrument_manager/help/instrument_manager.md]: done!

File: modules/instrument_manager/help/instrument_manager.md (hunk 1 above)
Comment (knoll-alex) [reply modules/instrument_manager/help/instrument_manager.md]: done

File: modules/instrument_manager/help/instrument_manager.md (hunk 2 above)
Comment (knoll-alex) [reply modules/instrument_manager/help/instrument_manager.md]: done

**Comment by knoll-alex** [comment]
@christinerogers I made updates to the text based on your requests. Here's a new screenshot. Note that `this` format doesn't render in the the help text - do you have a suggestion for how we should display those terms, such as `instrument_subsets` table? Would bold be fine?

Screenshot 2019-11-20 12 03 33

**Comment by christinerogers** [comment]
> @christinerogers I made updates to the text based on your requests. 
Looks great. thanks for the screenshot! 
I see a typo and no caps on a module name so i'll make a commit. 

> Note that `this` format doesn't render in the the help text - do you have a suggestion for how we should display those terms, such as `instrument_subsets` table? 

ah, good catch. 
Consistency is the first priority, in case you find other table names with a specific formatting (though i doubt it, back-end table names shouldn't be referenced in many front-end modules at all)
Otherwise i think `instrument_subsets` and `test_names` look fine as they are.  
(Bold is almost too attention-catching for table names, I don't have a strong preference for it.)

**Comment by christinerogers** [comment]
@driusan  ready to merge, it would seem

**Pull request merged**